dist/component-temp.css

```diff
@@ -0,0 +1,90 @@
+/*
+Error: Undefined variable: "$vimed-blue-darkened".
```

Comment:
Seems like a typo somewhere is causing a build error?

Reply:
I corrected it and rebuilt, this could have been residual

Reply:
looks like its just a log, not critical, will remove and add.